Source/Meadow.Foundation.Peripherals/Sensors.Moisture.Capacitive/Samples/Capacitive_Sample/MeadowApp.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Sensors.Moisture.Capacitive_Sample
 {
-    public class MeadowApp : App<F7FeatherV2>
+    public class MeadowApp : App<F7FeatherV1>
```

Comment:
Should we keep the sample on V2 for public use?

Reply:
good catch, fixed!